terraform/modules/ops/build.tf

```diff
@@ -3,7 +3,8 @@
 resource "google_cloudbuild_trigger" "api_push_to_main" {
   project        = var.project_id
   count          = var.setup_cd_system ? 1 : 0
-  name           = "api-push-to-main"
+  name           = "api-new-build"
```

Comment:
This name change goes from `<short component>-<triggering action>` to `<short component>-<build objective>`.  This seems good to me if we don't plan to expand scope of this trigger's goals.

I wonder if "new" really adds value over "api-build" but will follow your judgement.

Reply:
`new` doesn't add that much value but it's a good middle ground between `api-build` and `api-build-new-build` lol